sufia-models/app/models/concerns/sufia/works/generic_work.rb

```diff
@@ -8,4 +8,4 @@ module GenericWork
     include Sufia::GenericFile::Permissions
     include Sufia::GenericFile::Metadata
```

Comment:
@yinlinchen Sorry for the confusion.  Delete this line and include your new file.